src/main/java/org/apache/kafka/connect/redis/RedisPartialSyncWorker.java

```diff
@@ -34,15 +34,26 @@
 
     private static final Logger log = LoggerFactory.getLogger(RedisPartialSyncWorker.class);
 
-    private final RedisReplicator replicator;
+    private RedisReplicator replicator =null;
```

Comment:
formatting `replicator = null;`

Reply:
You removed final as well?

Reply:
It is because I needed to call the constructor after checking the config values.